sdk/src/dlob/DLOBApiClient.ts

```diff
@@ -0,0 +1,33 @@
+import fetch from 'node-fetch';
+import { DLOBOrdersCoder } from './DLOBOrders';
+import { DLOB } from './DLOB';
+
+type DLOBApiClientConfig = {
+	url: string;
+};
+
+export class DLOBApiClient {
+	url: string;
+	dlobCoder = DLOBOrdersCoder.create();
+	lastSeenDLOB: DLOB;
+	lastSeenSlot = 0;
+
+	constructor(config: DLOBApiClientConfig) {
+		this.url = config.url;
+	}
+
+	public async getDLOB(slot: number): Promise<DLOB> {
+		const r = await fetch(this.url);
+		const resp = await r.json();
```

Comment:
i think since `fetch` can fail, should check for `r.ok` here, and throw otherwise with an err message, otherwise .json will fail and give a cryptic stack trace

Reply:
if it does error, how should we bubble up the error for the DLOBSubscriber 🤔  should we add an error event to the DLOBSubscriber event emitter?